<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
alternate-background-color: rgb(255, 85, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Ajouter un produit </string>
    </property>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ID du produit </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>QTT disponible </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Catégorie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_valider">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>186</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_quitter">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>186</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prix">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_qtt">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_categorie">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Modifier les parametres d'un  produit :</string>
    </property>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Donnez le nouveau prix: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Donnez le nouveau id du produit:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bouton_modifier">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>186</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choisissez l'id original souhaité du produit: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_id">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_prix">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_List_ID">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load_List_ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_mod">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>611</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Affichage:</string>
    </property>
    <widget class="QListView" name="tab_produit_affcihage">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choisissez l'id du produit: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_quitter_aff">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>186</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="aff_id">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Confirmer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>521</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Supprimer un produit </string>
    </property>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Chosissez l'id du produit: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="bouton_supprimer">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>186</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="supp_id">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCRUD">
    <property name="title">
     <string>CRUD</string>
    </property>
   </widget>
   <addaction name="menuCRUD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
